main/src/library/Fixpoint/Ram/RamStmt.flix

```diff
@@ -14,8 +14,11 @@
  * limitations under the License.
  */
 
+use Fixpoint/Ast.Constraint;
+
 namespace Fixpoint/Ram {
     pub enum RamStmt[v] {
+        case Facts(Array[Constraint[v]])
```

Comment:
This could even be `HeadPredicate`, right?. We should consider that for the future.

Reply:
Indeed.